.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["react", "es2015"],
+  "presets": ["react", "es2015", "stage-2"],
```

Comment:
How about `` "presets": ["react", "es2015", "transform-object-rest-spread"], `` for now?